src/main/java/mr/springbootapi/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,34 @@
+package mr.springbootapi.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(ItemNotFoundException.class)
+    @ResponseBody
+    public ResponseEntity<String> handleItemNotFoundException(ItemNotFoundException ex) {
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(ex.getMessage());
+    }
+
+    @ExceptionHandler(InvalidPriceException.class)
+    @ResponseBody
+    public ResponseEntity<String> handleInvalidPriceException(InvalidPriceException ex) {
+        return ResponseEntity.status(HttpStatus.CONFLICT).body(ex.getMessage());
```

Comment:
HttpStatus.BAD_REQUEST is more suitable in this case

>The HyperText Transfer Protocol (HTTP) 400 Bad Request response status code indicates that the server cannot or will not process the request due to something that is perceived to be a client error (for example, malformed request syntax, invalid request message framing, or deceptive request routing).

>Warning: The client should not repeat this request without modification.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400